layouts/partials/article-meta.html

```diff
@@ -1,5 +1,7 @@
 			<p>
 				{{ if not (eq (.Date.Format "2006-01-02") "0001-01-01") }}<time>{{ .Date.Format "2006-01-02" }}</time>{{ end }}
+{{ if default true (index .Site.Params "showwordcount") }}
```

Comment:
my understanding is that `index` returns null if the key is not set, whereas directly referencing it will raise an error. I can change this though if I'm wrong.